File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.pagingQuery = function (page, size) {
        return $http.get('../brand/pagingQuery?pageNum=' + page +
            '&pageSize=' + size);
    };
    this.addBrand = function (entity) {
        return $http.post('../brand/addBrand',entity);
    };
    this.updateBrand = function (entity) {
        return $http.post('../brand/updateBrand',entity);
    };
    this.deleteSelected = function (selectIds) {
        return $http.get('../brand/deleteSelected?ids='+selectIds);
    };
    this.criteriaQuery = function (page, size, searchEntity) {
        return $http.post('../brand/criteriaQuery?pageNum=' + page +
            '&pageSize=' + size,searchEntity);
    };
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;

/**
 * 品牌接口
 * @Auther: invokersun
 * @Date: 2019/1/16   23:19
 * @Desription:
 */
public interface BrandService {
    //查询所有
    List<TbBrand> findAll();
    //分页查询
    PageResult pagingQuery(int pageNum,int pageSize);
    //增加品牌
    Result addBrand(TbBrand tbBrand);
    //修改品牌
    void updateBrand(TbBrand tbBrand);
    //通过ID查询
    TbBrand findOne(Long id);
    //删除选中项
    void deleteSlected(Long[] ids);
    //条件查询
    PageResult criteriaQuery(TbBrand tbBrand,int pageNum,int pageSize);
}
